config/routes.rb

```diff
@@ -10,5 +10,6 @@
 
   get '/orgs' => 'orgs#index', :as => :orgs
   get '/:org/repos' => 'repos#index', :as => :repos
-  get '/:org/:repo/issues(/:username)' => 'issues#index', :as => :issues
+  get '/:org/:repo/issues' => 'issues#index', :as => :issues
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.